libs/designsystem/src/lib/components/modal/services/alert.helper.ts

```diff
@@ -39,12 +40,7 @@ export class AlertHelper {
 
   private getOkBtnText(config: AlertConfig) {
```

Comment:
Consider renaming to reflect new property name: 
```suggestion
  private getOkBtn(config: AlertConfig) {
```

Reply:
I thought about this, but ended up not doing it because it actually returns text in the end (line 48-56). Thoughts on that?

Reply:
Renamed in d4fc2b3